site-src/guides/index.md

```diff
@@ -317,7 +354,7 @@ A cluster with:
    1. Uninstall the InferencePool, InferenceModel, and model server resources
 
       ```bash
-      kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/inferencepool-resources.yaml --ignore-not-found
+      helm uninstall vllm-llama3-8b-instruct
```

Comment:
nit: I know it wasn't introduced by your PR, but can you add cleanup also to sim-deployment?
in case someone decides to run the sim deployment it won't be cleaned.
```bash
kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/vllm/sim-deployment.yaml --ignore-not-found
```